README.md

```diff
@@ -51,15 +54,15 @@ package main
 
 import (
 	"fmt"
-
-	// Import go-elektra
 	"github.com/ElektraInitiative/go-elektra/kdb"
 )
 
 func main() {
+	// TODO API REVIEW: Why here CreateKeySet and not NewKeySet?
 	ks := kdb.CreateKeySet()
 	keyName := "/go-elektra"
 
+	// TODO API REVIEW: Separated New and Open necessary?
```

Comment:
I wanted `Open` to be part of the interface so I can mock it or add a different implementation, like one that would call the commandline instead of the c library.

Reply:
In this case we should add `defer handle.Close()`, you are right. We can't do this in the handlers though - since they have a longer lifespan than a single http request